kview/src/main/java/dev/ikm/komet/kview/mvvm/viewmodel/PatternViewModel.java

```diff
@@ -71,7 +71,7 @@ public PatternViewModel() {
                     .addProperty(STAMP_VIEW_MODEL, (ViewModel) null)
                     .addProperty(DEFINITION_VIEW_MODEL, (ViewModel) null)
                     .addProperty(FQN_DESCRIPTION_NAME, (DescrName) null)
-                    .addProperty(OTHER_NAMES, (List) null)
+                    .addProperty(OTHER_NAMES, (List<DescrName>) new ArrayList<DescrName>())
```

Comment:
Can you remove the cast?
The cast is used whenever the initial value is `null` to let Cognitive's view model know the data type for the property. If the value is a `List` the API will internally know to create an `ObservableList<DescrName>` for property `OTHER_NAMES`.

Reply:
Done.